pallets/grants/src/lib.rs

```diff
@@ -49,6 +49,7 @@ pub type BalanceOf<T> =
     <<T as Config>::Currency as Currency<<T as frame_system::Config>::AccountId>>::Balance;
 pub type VestingScheduleOf<T> =
     VestingSchedule<<T as frame_system::Config>::BlockNumber, BalanceOf<T>>;
+pub type VestingScheduleOfColl<T> = Vec<VestingScheduleOf<T>>;
```

Comment:
If possible, I think we should rename to something that is more meaningful. Maybe `VestingSchedulesOf<T>` or `ListVestingScheduleOf<T>` for instance?

Reply:
Sure Eliott will do that :+1: